lib/mix/tasks/gh_pages_docs_new.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Mix.Tasks.GhPagesDocs.New do
+  use Mix.Task
+
+  @moduledoc """
+  Task for adding a new documentation version published on GH Pages.
+  The version (format: vX.Y.Z) should be passed as the argument for the task.
```

Comment:
Is this `v` prefix required? Git tags don't contain that prefix and I find it a bit confusing that the software version is just an `X.Y.Z` while the doc version needs to be prefixed with `v`.

Reply:
Oh, and how about the `latest` or `master` documentation without a tag yet?

Reply:
I checked in few main Elixir projects' documentation on hexdocs and `vX.Y.Z` seems to be the right convention.

Also, I changed the task implementation and the argument is not needed anymore as we have access to the project current version which is also the latest, i.e. now we have `2.1.0-alpha.1` till `2.1.0` is released.